mass_mailing_custom_unsubscribe/README.rst

```diff
@@ -0,0 +1,81 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Customizable unsubscribe link on mass mailing emails
+====================================================
+
+With this module you can set a custom unsubscribe link append at bottom of mass
+mailing emails.
+
+
+Installation
+============
+
+Nothing special to do
+
+
+Configuration
+=============
+
+To configure unsubscribe label go to Setting > Technical > Parameters > System parameters
+and add a 'mass_mailing.unsubscribe.label' parameter with html to set at bottom
+of mass emailing emails. Including '%(url)s' variable where unsubscribe link
+
+For example:
+```
+<small>You can unsubscribe <a href="%(url)s">here</a></small>
+
+```
+
+Additionally, you can disable this link if you set this parameter to 'False'
```

Comment:
Mention that if the parameter is empty, a default text and url is used. This one is translatable.